semgrep-core/tests/rules/taint_param_default.yaml

```diff
@@ -0,0 +1,29 @@
+rules:
+  - id: request-session-with-http
+    options:
+      symbolic_propagation: true
+    mode: taint
+    pattern-sources:
+      - patterns:
+          - pattern: $URL
```

Comment:
hmm should be "$URL"?
or maybe pattern: |
  "$URL"
(I hate yaml).

Reply:
Good point, I guess they may be using `metavariable-pattern` instead of `metavariable-regex` because then they can use `pattern-not-regex` ? cc @minusworld